grafana-plugin/src/plugin/GrafanaPluginRootPage.tsx

```diff
@@ -206,4 +201,13 @@ export const Root = observer((props: AppRootProps) => {
       </div>
     </DefaultPageLayout>
   );
+
+  async function runQueuedUpdateData() {
+    try {
+      await store.updateBasicData();
+      setBasicDataLoaded(true);
+    } catch {
+      setTimeout(runQueuedUpdateData, 1000);
```

Comment:
if this is failing it will loop infinitely, I would consider adding a max number of retries before failing and showing the user some error message